Update RoleNORManager.md (#2631)
OBOFoundry/OBOFoundry.github.io: docs/roles/RoleNORManager.md

When the ontology is accepted, the OBO NOR reviewer needs to make sure that the accepted ontology is removed from the OBO NOR Dashboard.

Note 29.11.2022: There is a [work in progress](https://github.com/OBOFoundry/obo-nor.github.io/pull/23) which can deal with this step automatically, but the responsibility still stays with the OBO NOR Manager.

### Maintaining the OBO Duty Rotation Table

> The OBO Duty Rotation Table is a Google Doc available [here](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit).

Reviewer: mention that it is editable by everyone on the OBO Ops committee?

> The OBO Duty Rotation Table is a Google Doc available [here](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit).

Reviewer: ```suggestion
The OBO Duty Rotation Table is a Google spreadsheet available [here](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit).
```

> When a new member is added to OBO Foundry, the OBO NOR reviewer should add their name to this table. Likewise, former members should be removed from the table. The soure of truth for OBO Foundry member is the file [operations.yml](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/operations.yml).

Reviewer: ```suggestion
When a new member is added to OBO Foundry, the OBO NOR reviewer should add their name to this table. Likewise, former members should be removed from the table. The list of current OBO Foundry members is maintained in the file [operations.yml](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/operations.yml).
```

> When a new member is added to OBO Foundry, the OBO NOR reviewer should add their name to this table. Likewise, former members should be removed from the table. The list of current OBO Foundry members is maintained in the file [operations.yml](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/operations.yml).

Reviewer: ```suggestion
When a new member is added to the OBO Operations Committee, the OBO NOR reviewer should add the new member's name to this table. Similarly, former members should be removed from the table. The list of current OBO Operations members is maintained in the file [operations.yml](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/_data/operations.yml).
```

> In addition, when a new ontology has been added, it should be documented to the 'Ontology Review Rotation' sheet in the same document.

Reviewer: ```suggestion
In addition, when a new ontology has been submitted, it should be documented in the 'Ontology Review Rotation' sheet in the same document.
```

> In addition, when a new ontology has been submitted, it should be documented in the 'Ontology Review Rotation' sheet in the same document.

Reviewer: ```suggestion
In addition, when a new ontology has been submitted, it should be added to the ['Ontology Reviewers' tab](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit?gid=1683009411#gid=1683009411) in the OBO Duty Rotation spreadsheet.
```

Reviewer: Sorry, I missed this in my first edit.
